static/css/app.css

```diff
@@ -214,7 +214,7 @@ h1, h2, h3, h4, h5, h6 {
 }
 
 #message-area.hidden {
-  transform: translateY(10rem);
+  transform: translateY(100%) translateY(4rem);
```

Comment:
Is this what you meant? two translateYs?

Reply:
Yes, though I could do `translateY(calc(100% + 4rem))` instead. This just felt clearer at the time

Reply:
Huh, I didn't even know that worked! calc might be more readable, but it's a minor point.